### PR TITLE
WIP: More autocompletion

### PR DESCRIPTION
**Pull request by justinsb**
Nouns

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1112)

### CODE REVIEW

File: cmd/kops/toolbox_completion.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+func RunToolboxCompletion(f *util.Factory, cmd *cobra.Command, args []string, out io.Writer, options *ToolboxCompletionOptions) error {
+	if len(args) == 0 {
+		return fmt.Errorf("No args were provided (this command is normally called from a bash completion script)")
+	}
+
+	lastCommand := args[0]
+	switch lastCommand {
+	case "kops_get_clusters":
+		fmt.Fprintf(out, "kopeio.awsdata.com\n")
```
Comment (chrislovecnm) [inline cmd/kops/toolbox_completion.go]: Not sure what you are doing

**Comment by kris-nova** [comment]
@chrislovecnm all yours - you know more about the autocomplete things

**Comment by chrislovecnm** [comment]
@justinsb still WIP?

**Comment by justinsb** [comment]
Very much still a WIP.  Just thought I would occasionally chip away at it

**Comment by chrislovecnm** [comment]
cc: @frodopwns

**Comment by kris-nova** [comment]
Hey @justinsb 

How is the chipping away going here? Wondering if you wouldn't mind checking out the conflicts, and maybe updating the description here with a sentence or two? Seems a bit terse 😉

**Comment by chrislovecnm** [comment]
What is the status on this?

**Pull request closed without merging**